code/modules/clothing/gloves/colored_gloves.dm

```diff
@@ -166,19 +154,17 @@
 	item_state = "browngloves"
 	item_color="brown"
 
-/obj/item/clothing/gloves/color/brown/cargo
-	item_color = "cargo"				//Exists for washing machines. Is not different from brown gloves in any way.
-
 /obj/item/clothing/gloves/color/latex
 	name = "latex gloves"
 	desc = "Cheap sterile gloves made from latex."
 	icon_state = "latex"
 	item_state = "lgloves"
 	siemens_coefficient = 0.30
 	permeability_coefficient = 0.01
-	item_color="white"
+	item_color = "white"
 	transfer_prints = TRUE
 	resistance_flags = NONE
+	dyeable = FALSE // can't dye latex
```

Comment:
I don't see why not. If you do want to keep this, please add it to the changelog and PR description.

Reply:
removed in next commit